plugins/tools/app/javascript/widgets/castellum/components/error_list.jsx

```diff
@@ -118,7 +131,7 @@ export default class Loader extends React.Component {
         pageSize={6}
       >
         {data.map((error, idx) => (
-          <ErrorRow key={`error${idx}`} error={error} />
+          <ErrorRow key={`error${idx}`} error={error} action={{label: "Clear", fn: this.onAction}}/>
```

Comment:
```suggestion
          <ErrorRow key={`error${idx}`} error={error} action={{label: "Mark as resolved", fn: this.onAction}}/>
```
"Clear" sounds like it's performing an action on the asset, instead of merely updating the presented state.

Reply:
I checked how the longer name behaves in conjunction with the column size and it seems to be okay for the other content to still fit in the row.